### ISSUE TITLE
Label remove button for students

### ISSUE THREAD

**Issue by bwreid** [WikiEducationFoundation/WikiEduDashboard#2531]
![image](https://user-images.githubusercontent.com/1316902/53433636-9eeb7c80-39a9-11e9-9d8a-5656e0571a83.png)

The remove button (-) for students is unlabeled and therefore inaccessible by screen readers. Look for `removeButton` in AssignButton, EnrollButton, TagEditable, and CampaignEditable and add functionality so that these buttons describe what they do.

**Comment by helenatxu** [comment]
Hello @bwreid! I am an applicant for the Outreachy internship, can I take this issue for my first contribution? 

Do you think it would be a good start? I already have some experience with React.

**Comment by ragesoss** [comment]
@helenatxu go for it.

**Comment by mariahajmal** [comment]
@helenatxu hey, are you working on this?

**Comment by ragesoss** [comment]
No, they aren't working on it anymore I don't think.

**Comment by mariahajmal** [comment]
@ragesoss thanks. Do I need to add a label(Remove) before (-) ?

![Remove label](https://user-images.githubusercontent.com/23306376/54451459-b5991e00-4774-11e9-90c7-480002c6bca8.png)

> No, they aren't working on it anymore I don't think.

**Comment by ragesoss** [comment]
The button itself should have that label (which should be visible; it's purpose is mainly for screen readers).
